### PR TITLE
Inject CozyClient in Bar

### PR DESCRIPTION
**Pull request by gregorylegarec**
Some custom versions of the CozyBar are needing a injected CozyClient.

### CODE REVIEW

**Comment by ptbrowne** [comment]
> Some custom versions of the CozyBar

Is banks using a custom version ? I do not understand.

File: src/main.jsx [hunk 1]
```diff
@@ -66,6 +66,7 @@ const setupApp = async persistedState => {
   if (__TARGET__ !== 'mobile') {
     cozy.bar.init({
       appName: data.cozyAppName,
+      cozyClient: client,
```
Comment (drazik) [inline src/main.jsx]: We want to do this only for web app?

Comment (gregorylegarec) [reply src/main.jsx]: No, is there somewhere else to inject cozy Client ?

Comment (gregorylegarec) [reply src/main.jsx]: Thanks for pointing out this, I've just updated the commit.

Comment (drazik) [reply src/main.jsx]: Yes, here: https://github.com/cozy/cozy-banks/blob/master/src/ducks/mobile/utils.js#L24

**Pull request merged**